### PR TITLE
New Blog for Release 0.6.0

### PR DESCRIPTION
**Pull request by aravindavk**
Signed-off-by: Aravinda Vishwanathapura <aravinda@kadalu.io>

### CODE REVIEW

File: _posts/2020-03-01-kadalu-storage-0.6-and-future.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+## Replica 2 support for general purpose
+
+Kadalu Team will work on extending Replica 2 support for general
+purpose using the thin arbiter feature of GlusterFS. Follow the steps
+mentioned in the below tweet to setup the Kadalu Tiebreaker container
+yourself.
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Two options made available from us:<br>1. use `<a href="https://t.co/VvuopJSQSn">https://t.co/VvuopJSQSn</a>:/mnt` as the thin-arbiter brick for your volume.<br>Or :<br>2. `docker run --privileged -p 24007:24007 -v ${hostbrickdir}:/mnt/brick <a href="https://t.co/3NV9VrPu5r">https://t.co/3NV9VrPu5r</a>`</p>&mdash; Kadalu (@kadaluIO) <a href="https://twitter.com/kadaluIO/status/1225677569439125504?ref_src=twsrc%5Etfw">February 7, 2020</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
```
Comment (amarts) [inline _posts/2020-03-01-kadalu-storage-0.6-and-future.md:65]: Does it render well? Just looking at the preview didn't give me a great look. If there are issues, we may have to 'natively' format the content, with just a link to tweet.

**Review by amarts — APPROVED** [review]

File: _posts/2020-03-01-kadalu-storage-0.6-and-future.md (hunk 1 above)
Comment (aravindavk) [reply _posts/2020-03-01-kadalu-storage-0.6-and-future.md:65]: Github preview doesn't allow Javascript embed, but rendered site displayes properly. This code is copied from here https://publish.twitter.com/?query=https%3A%2F%2Ftwitter.com%2FkadaluIO%2Fstatus%2F1225677569439125504&widget=Tweet

Screenshot_2020-03-01 Kadalu Storage 0 6 and Future

**Comment by aravindavk** [comment]
Added two new things

Under "New Features":

* Kadalu base container image upgraded to Fedora 31 and Gluster
  version in server and CSI container images to 7.3 ([PR #198](https://github.com/kadalu/kadalu/pull/198))


Under "Important bug fixes":

* Fixed issue in kubectl kadalu plugin while parsing arguments related
  to external storage and Tiebreaker node ([Issue #199](https://github.com/kadalu/kadalu/issues/199)).

**Comment by sankarshanmukhopadhyay** [comment]
LGTM

**Pull request merged**